org.eclipse.xtext.full.releng/pom.xml

```diff
@@ -1,15 +1,16 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>org.eclipse.xtext</groupId>
-		<artifactId>xtext-parent</artifactId>
-		<version>2.31.0-SNAPSHOT</version>
-	</parent>
+	<groupId>org.eclipse.xtext</groupId>
```

Comment:
what is the background behind this change?

Reply:
The releng projects act only as aggregators: we don't publish them, and there's no need to make them inherit from the parent POM. This allows updating versions without problems due to possible cycles since they are independent of the versions of our actual POMs.